pkg/queue/instance.go

```diff
@@ -27,24 +28,32 @@ import (
 type Task func() error
 
 // Instance of work tickets processed using a rate-limiting loop
-type Instance interface {
```

Comment:
Doesn't seem like much extra work to add the functionality to delayqueue

Reply:
It was hard since there are multiple workers